docs/transactions.md

```diff
@@ -0,0 +1,113 @@
+# Transactions v2
+
+## URL
+
+### Account transactions
+
+All Transactions from a given Account:
+`https://rest.opencollective.com/v2/{slug}/transactions.csv`
+
+E.g. https://rest.opencollective.com/v2/babel/transactions.csv
+
+### Host transactions
+
+All Transactions accounted by a given Fiscal Host:
+`https://rest.opencollective.com/v2/{slug}/hostTransactions.csv`
+
+E.g. https://rest.opencollective.com/v2/opensource/hostTransactions.csv
+
+### Parameters
+
+- `limit`: default is `1000`, maximum is `10000`
+- `offset`: default is 0
+- `type`: `CREDIT` or `DEBIT`
+- `kind`: comma separated list of `KIND`s
+- `dateFrom`: transactions after UTC date
+- `dateTo`: transactions before UTC date
+- `minAmount`: transactions more than the amount (warning, in cents!)
+- `maxAmount`: transactions less than the amount (warning, in cents!)
+- `includeIncognitoTransactions`: include incognito transactions made by the account (only for authenticated user)
+- `includeChildrenTransactions`: include transactions by children of the account (projects and events)
+- `includeGiftCardTransactions`: include transactions with Gift Cards issued by the account
+
+### Authentication
+
+Create an Api Key and pass it in the URL parameters as `apiKey`.
+
+### Tips
+
+- Replace `.csv` with `.txt` to view in plain text in the browser instead of downloading
+
+## Fields
+
+| Name                 | GraphQL v2                       | description                                               | default |
```

Comment:
When I first saw "default" in the table, I thought it meant "Default value" and that confused me. Maybe we can clarify that by renaming the column to something like "Included by default"